src/common/hooks/useResolveGateway.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { CompanyGateway } from '../interfaces/company-gateway';
+import { useCompanyGatewaysQuery } from '../queries/company-gateways';
+
+export const useResolveGateway = () => {
```

Comment:
Please follow the conventions of the project and use `function $name`.

Reply:
Also, can you explain why we use these fns? Looks to me they're simple .find, why need for _another_ method?

Reply:
Changed! Sure, the purpose of this hook is the same as for all the others I've created. I've explained its actual purpose in this comment: `https://github.com/invoiceninja/ui/pull/692#discussion_r1317307121`, so to avoid duplicating it, please refer to that one. Let me know your thoughts.